src/Datagrid/Pager.php

```diff
@@ -50,7 +106,7 @@ public function init()
     {
         $this->resetIterator();
 
-        $this->setNbResults($this->computeNbResult());
+        $this->setResultsCount($this->computeNbResult('sonata_deprecation_mute'));
```

Comment:
```suggestion
        $this->setResultsCount($this-> computeResultsCount());
```
Can we use the new method here?

Reply:
Someone might have overridden the old one...

Reply:
hmmm in that case I guess we should call `setNbResults` instead of `setResultsCount` and leave the old calls with `'sonata_deprecation_mute'`?

Reply:
`setResultsCount` also calls `setNbResults`: https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/501/files#diff-f13ffebef517d741bc6f655c9465d29a5154a3c9a1ac5746acb85654dfc606f4R56